deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -78,5 +103,15 @@ export class NetworkStack extends Stack {
             });
         }
 
+        if (props.targetClusterEndpoint) {
+            const formattedClusterEndpoint = NetworkStack.validateAndReturnFormattedHttpURL(props.targetClusterEndpoint)
+            const deployId = props.addOnMigrationDeployId ? props.addOnMigrationDeployId : props.defaultDeployId
```

Comment:
What is an addOnMigrationDeployId?  The scalar (rather than collection) type is confusing me.